Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -2600,7 +2600,7 @@ void MediaPlayerPrivateGStreamer::updateStates()
                 invalidateCachedPosition();
                 GST_DEBUG_OBJECT(pipeline(), "[Seek] seeking to %s failed", toString(m_seekTarget.time).utf8().data());
             }
-        } else if (m_isSeeking && !(state == GST_STATE_PLAYING && pending == GST_STATE_PAUSED))
+        } else if (!isMediaSource() && m_isSeeking && !(state == GST_STATE_PLAYING && pending == GST_STATE_PAUSED))
```

Comment:
MediaPlayerPrivateGStreamerMSE overrides `updateStates()` with its own independent implementation.

Reply:
Noted... Anyway this PR is bit-rotten already after https://commits.webkit.org/267814@main